### PR TITLE
Validate index

### LINKED ISSUE #208: Validate HTML
### Evidence / Screenshot (if possible)
image

### Steps to Reproduce

1. check here https://validator.w3.org/nu/?doc=https%3A%2F%2Fprojectsakura.xyz%2F

* Actual: 
* Expected: 

### Details

- **Browser type/version?** 
- **Operating system?**

### PR DESCRIPTION
**Pull request by tanishq-arya**
Closes #208 

Removed center tags and errors.
Added center, center-post classes to achieve center tag functionality.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -303,7 +300,7 @@ <h1>
     <section class="single-page scrollblock" id="screenshots">
         <div class="">
             <div class="align">
-                <img alt="download-icon" style="size:10%" src="image/screenshot.png">
```
Comment (ArunTeltia) [inline index.html:306]: can i get the image of screenshots whether they are looking same or not ?
If yes great work finding this and removing it

File: index.html [hunk 2]
```diff
@@ -354,7 +349,7 @@ <h1 id="folio-headline">
     <section class="single-page scrollblock" id="features">
         <div class="container">
             <div class="align">
-                <img alt="download-icon" style="size:10%" src="image/feature.png">
```
Comment (ArunTeltia) [inline index.html:357]: same as above

File: index.html [hunk 3]
```diff
@@ -1125,12 +1086,11 @@ <h2>
...
         crossorigin="anonymous"></script>
     <!-- Loading the javaScript at the end of the page -->
-    <script src="js/bootstrap.js" type="text/javascript">
-    </script>
-    <script src="js/navScroll.js" type="text/javaScript"></script>
+    <script src="js/bootstrap.js"></script>
+    <script src="js/navScroll.js"></script>
 
     <!-- OWl Carousel -->
-    <script type="text/javascript">
```
Comment (ArunTeltia) [inline index.html:1133]: is removing type fixes something?

**Comment by tanishq-arya** [comment]
Screenshot (1058)
Screenshot (1056)
Screenshot (1057)

File: index.html (hunk 3 above)
Comment (tanishq-arya) [reply index.html:1133]: It was giving a warning that script doesn't require type in HTML5.

File: index.html (hunk 1 above)
Comment (tanishq-arya) [reply index.html:306]: Size is no longer supported I removed a it and it works.

Comment (tanishq-arya) [reply index.html:306]: Size is not supported now

**Comment by ArunTeltia** [comment]
Mobile view as well please

File: index.html (hunk 1 above)
Comment (ArunTeltia) [reply index.html:306]: Is it? I guess, I am not aware of it, it looks good to me just I am a bit paranoid for building something that will break the page

**Comment by tanishq-arya** [comment]
Screenshot (1059)
Screenshot (1060)
Screenshot (1061)
Screenshot (1062)
Screenshot (1063)
Screenshot (1064)

**Comment by tanishq-arya** [comment]
Fixing the last one now.

**Comment by tanishq-arya** [comment]
image

**Comment by ArunTeltia** [comment]
Looks great to me @LordShenron , Merge

**Comment by tanishq-arya** [comment]
@LordShenron This is a level 3 issue. Please change the tags.

**Comment by ArunTeltia** [comment]
Yup changed @tanishq-arya

**Pull request merged**